snuba/query/expressions.py

```diff
@@ -163,9 +163,7 @@ def accept(self, visitor: ExpressionVisitor[TVisited]) -> TVisited:
 
     def transform(self, func: Callable[[Expression], Expression]) -> Expression:
         transformed = replace(
-            self,
-            column=self.column.transform(func),
-            key=self.key.transform(func),
+            self, column=self.column.transform(func), key=self.key.transform(func),
```

Comment:
do you mean reformatting the file? This happened because I had made a change in an earlier version and black decided to reformat. Since it was only one statement I did not bother reverting it.

Reply:
Yeah, but mostly how/why these changes that were okay at one point aren't now (presuming they were and didn't just slip in.)